tests/components/ring/test_number.py

```diff
@@ -0,0 +1,94 @@
+"""The tests for the Ring switch platform."""
+
+import pytest
+
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+
+from .common import setup_platform
+
+
+@pytest.mark.parametrize(
+    ("entity", "unique_id"),
+    [
+        ("number.downstairs_volume", "123456-volume"),
+        ("number.front_door_volume", "987654-volume"),
+        ("number.ingress_doorbell_volume", "185036587-doorbell_volume"),
+        ("number.ingress_mic_volume", "185036587-mic_volume"),
+        ("number.ingress_voice_volume", "185036587-voice_volume"),
+    ],
+)
+async def test_entity_registry(
+    hass: HomeAssistant,
+    entity_registry: er.EntityRegistry,
+    mock_ring_client,
+    entity,
+    unique_id,
+) -> None:
+    """Tests that the devices are registered in the entity registry."""
+    await setup_platform(hass, Platform.NUMBER)
+
+    entry = entity_registry.async_get(entity)
+    assert entry is not None and entry.unique_id == unique_id
+
+
+@pytest.mark.parametrize(
+    ("entity", "initial_state"),
+    [
+        ("number.downstairs_volume", "2.0"),
+        ("number.front_door_volume", "1.0"),
+        ("number.ingress_doorbell_volume", "8.0"),
+        ("number.ingress_mic_volume", "11.0"),
+        ("number.ingress_voice_volume", "11.0"),
+    ],
+)
+async def test_initial_state(
```

Comment:
Ugh, that appears to be a bit too much black magic for me :sweat_smile: Honestly, I don't understand how it works. I tried copying the `test_states()` function (obviously replacing `SIREN` by `NUMBER`), but the command you suggested fails with `FAILED tests/components/ring/test_number.py - Failed: In test_states: function uses no argument 'entity_id'`. Since I really don't know what I'm doing it's hard for me to work eith this error message.

Can you provide a another pointer maybe? Or maybe this could be cleaned up later?

Reply:
It should work fine. Maybe you just need to recompile your translations:
```python3 -m script.translations develop --all```

Reply:
Ah well, I just forgot to uncomment the `@pytest.mark.parametrize` decorator of the old function :sweat_smile:  Now the test passes, but tbh I can't really tell if the test makes sense or not. Here's what I came up with: 258c02e007

Reply:
You've forgotten to include the generated `test_number.ambr` file

Reply:
Oops. d8999541ab12fefca7631f415ac1f0a774829a11